src/NodeManipulator/ClassConstManipulator.php

```diff
@@ -25,8 +26,8 @@ public function __construct(
 
     public function hasClassConstFetch(ClassConst $classConst, ClassReflection $classReflection): bool
     {
-        $class = $this->betterNodeFinder->findParentType($classConst, Class_::class);
-        if (! $class instanceof Class_) {
+        $class = $this->betterNodeFinder->findParentByTypes($classConst, [Class_::class, Enum_::class]);
+        if ($class === null) {
             return false;
```

Comment:
`return true` can be used since it can be trait, which consumeable by other class:

```suggestion
            return true;
```

Reply:
see https://3v4l.org/pJo9C#v8.2.0 on php 8.2